### DISCUSSION TITLE
do not work for game window which even got focus.....

### DISCUSSION THREAD

**Discussion by chlfong** [taojy123/KeymouseGo#139] [Q&A] [answered]
do not work for game window which even got focus.....

**Comment by ZutJoe** [comment]
You can translate to see https://github.com/taojy123/KeymouseGo/issues/121#issuecomment-1146737204 and We don't specifically adapt the game.

Maybe you can look for professional game aids

**Comment by OKOtohime** [comment, marked as answer]
Hi, currently this program is based on Windows message mechanism, which is at system layer. For some games, mouse/keyboard operation may be captured on driver layer via DirectX, etc, which makes this program invalid. I'm afraid to inform you that currently this program cannot support games well as it requires deeper knowledge into message mechanism and replacement of core library. 😕
